sapio-data-repr/src/lib.rs

```diff
@@ -16,10 +16,20 @@ struct SapioModuleBoundarySerializer<W> {
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct SapioModuleBoundaryRepr(Value);
+impl SapioModuleBoundaryRepr {
```

Comment:
these functions seem to be dependent on SapioModuleBoundryRepr having a slice internally.

Reply:
These have been removed in a later iteration of this PR.